### ISSUE TITLE
Armor System

### ISSUE THREAD

**Issue by NicoProgrammer1** [NicoProgrammer1/TextbasedGame1#3]
We want to figure out how we can attach objects to the person class through which attributes of the person class may change.

**Comment by ghost** [comment]
Yeah it's very basic right now I think we may need to have three types of item
**Wearables**
There should be wearables that have a subtype depending on what body part they protect
**Perishables**: food,  potions
**Weapons:**
Swords Axes etc
Should be able to be used  to attack opponents
